assets/airports/kcvg.json

```diff
@@ -0,0 +1,6856 @@
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "KCVG",
+            "category": "arrival",
+            "route": "JOT.CEGRM4.KCVG",
+            "altitude": 29000,
+            "speed": 300,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["dal", 51],
+                ["aal", 17],
+                ["ual", 11],
+                ["aca", 2],
+                ["swa", 1]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "KCVG",
+            "category": "arrival",
+            "route": "COLNS.GAVNN4.KCVG",
+            "altitude": 20000,
+            "speed": 300,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["dal", 51],
+                ["aal", 17],
+                ["ual", 11],
+                ["aca", 2],
+                ["swa", 1]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "KCVG",
+            "category": "arrival",
+            "route": "HNN.GAVNN4.KCVG",
+            "altitude": 20000,
+            "speed": 300,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["dal", 51],
+                ["aal", 17],
+                ["ual", 11],
+                ["aca", 2],
+                ["swa", 1]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "KCVG",
+            "category": "arrival",
+            "route": "ARTUR.JAKIE4.KCVG",
+            "altitude": 20000,
+            "speed": 300,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["dal", 51],
+                ["aal", 17],
+                ["ual", 11],
+                ["aca", 2],
+                ["swa", 1]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "KCVG",
+            "category": "arrival",
+            "route": "CUBIM.JAKIE4.KCVG",
+            "altitude": 20000,
+            "speed": 300,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["dal", 51],
+                ["aal", 17],
+                ["ual", 11],
+                ["aca", 2],
+                ["swa", 1]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "KCVG",
+            "category": "arrival",
+            "route": "DOLLI.JAKIE4.KCVG",
+            "altitude": 20000,
+            "speed": 300,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["dal", 51],
+                ["aal", 17],
+                ["ual", 11],
+                ["aca", 2],
+                ["swa", 1]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "KCVG",
+            "category": "arrival",
+            "route": "VXV.JAKIE4.KCVG",
```

Comment:
Remove three of the spawnPatterns for the JAKIE arrival as they will meet at some point and cause seperation issues.